includes/functions.php

```diff
@@ -1497,6 +1547,33 @@ function tsml_import_buffer_set($meetings, $data_source_url = null, $data_source
     $indexes_to_remove = [];
 
     for ($i = 0; $i < count($meetings); $i++) {
+
```

Comment:
Since Google spreadsheets can have entirely non-unique slugs or none at all, so we can't really tie incoming meetings to ones in the DB. So just made certain in this step I had a semi-reliable, unique slug, so I could better connect incoming to ones we have.

Reply:
That was the reason we decided not to import the google spreadsheets as updateable data sources. Are we now doing so?

Reply:
Yep, we figured really to treat all data sources the same way for simplification. The sanitize method does still call our existing speadsheet cleanup function, but otherwise this does an adequate job of coming up with consistent slugs to use, and with that we can compare old to new.

Reply:
Sounds good...